packages/theme/functions.php

```diff
@@ -209,11 +213,11 @@ function register_rest_media()
 };
 function get_rest_media($object, $field_name, $request)
 {
-    if($object['featured_media']) {
-        $media = wp_get_attachment_image_src($object['featured_media'], 'large')[0];
-        return $media;
+    if (!array_key_exists('featured_media', $object) || !$object['featured_media']) {
```

Comment:
a to nie jest tak, że jak klucza nie ma to zwróci coś _falsy_?

Reply:
Bo w sumie nie widzę innych zmian de facto

Reply:
W php7.4 rzuca notice'a (oczywiście o ile na serwerze jest włączone pokazywanie błędów). Te notice'y uniemożliwiają sparsowanie JSONa w API.